charts/veritable-cloudagent/templates/deployment.yaml

```diff
@@ -173,7 +173,7 @@ spec:
             {{- end }}
             {{- if .Values.webhookUrl }}
             - name: AFJ_REST_WEBHOOK_URL
-              value: {{ .Values.webhookUrl | quote }}
+              value: {{ join " " .Values.webhookUrl | quote }}
```

Comment:
https://github.com/digicatapult/helm-charts/blob/8c53c161913deeda86c996a6d14f9757f4c16032/charts/veritable-ui/templates/secret-session-cookie.yaml#L17

minus the `b64encode`